### PR TITLE
Add basic UI

### PR DESCRIPTION
**Pull request by jmoseka**
In this PR, the following requirements are met

### Project requirements
- Watch the video with the UI example again (you have seen it while reading the Sneak Peek).
    - [UI example](https://www.youtube.com/watch?v=vkkgrhD6aXQ)
- Your console app should behave in the same way is in the example.
- Create a `app.rb` file that will serve as your console app entry-point. It should have methods that do the following:
  - List all books.
  - List all people.
  - Create a person (teacher or student, not a plain `Person`).
  - Create a book.
  - Create a rental.
  - List all rentals for a given person id.
- In your `main.rb` define the entry point, this will be a method called `main` that is invoked at the end of your file. This method should do the following:
  - Present the user with a list of options to perform.
  - Lets users choose an option.
  - If needed, ask for parameters for the option.
  - Have a way to quit the app.

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Has parent permission? [Y/N]: '
+    parent_permission = gets.chomp != 'n'
+    st = Student.new(age, name, parent_permission)
+    @persons << st
+    puts 'create student'
+  end
+
```
Comment (rotimiazeez) [inline app.rb]: - The below error was encountered when creating a student, kindly fix it👍🏽
Screenshot from 2022-07-23 19-11-22

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+  def create_teacher
+    print 'Age: '
+    age = gets.chomp
+    print 'Name: '
+    name = gets.chomp
+    print 'Specialization: '
+    specialization = gets.chomp
+    @persons << Teacher.new(age, name, specialization)
+    puts 'create teacher'
+  end
```
Comment (rotimiazeez) [inline app.rb]: - The below error was encountered when creating a teacher, kindly fix it👍🏽
Screenshot from 2022-07-23 19-12-28

File: app.rb [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    puts options
+    gets.chomp.to_i
+  end
+
+  def books_list
+    if @books.length.positive?
+      @books.each do |book|
+        puts " Title: #{book.title}, Author: #{book.author}"
+      end
+    else
```
Comment (rotimiazeez) [inline app.rb]: - The below error was encountered when listing all books, kindly fix it 👍🏽
Screenshot from 2022-07-23 19-42-21

**Review by rotimiazeez — CHANGES_REQUESTED** [review]
# Hi @jmoseka  👏🏽, 
## Good job so far! 👍🏽 
### Summary ♨️ 
- No Linter error ✔️ 
- Good and descriptive PR 💯
- Correct git flow followed 🥇
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## Highlights


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: app.rb (hunk 1 above)
Comment (DeliceLydia) [reply app.rb]: This was addressed!

File: app.rb (hunk 2 above)
Comment (DeliceLydia) [reply app.rb]: addressed

File: rental.rb [hunk 4]
```diff
@@ -17,8 +17,6 @@ def initialize(date, book, person)
     person.rental << self
   end
```
Comment (DeliceLydia) [inline rental.rb:19]: - [x] I am not able to create a `rental` number 5
rental

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @jmoseka, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jmoseka** [comment]
Issue fixed. One needs at least a list of person and book in order to rent. This demo shows rent created and list of rented books
Capture

**Review by uwadonat — APPROVED** [review]
## Hi @jmoseka,

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 

# Status: Approved ✔️✔️✔️

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100%.
Please, remember to tag me @uwadonat in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Comment by jmoseka** [comment]
@rotimiazeez @DeliceLydia @uwadonat  Thank you so much for your reviews

**Pull request merged**